### PR TITLE
Package prbnmcn-gnuplot.0.0.5

### PR DESCRIPTION
**Pull request by igarnier**
### `prbnmcn-gnuplot.0.0.5`
Declarative generation of gnuplot scripts
Declarative generation of gnuplot scripts

---
* Homepage: http://github.com/igarnier/prbnmcn-gnuplot
* Source repo: git+https://github.com/igarnier/prbnmcn-gnuplot
* Bug tracker: http://github.com/igarnier/prbnmcn-gnuplot

---
## 0.0.5
- Remove calls to `Unix.fork`, making the library multicore-friendly
- Expose implementation of `rN` types as tuples of floats
- Breaking: remove `silent` optional argument
- Breaking: remove now useless `tupN` functions

## 0.0.4
- add `xrange, yrange, zrange` options
- Add `get_targets` to get all available terminal types.
- Fix error in README.md example (report by ego@anentropic.com)

## 0.0.3
- Add circle plots
- Add `exec_detach` target, allows to plot several windows.
- Add `silent` optional argument, allows to silence gnuplot errors

## 0.0.2
- Add facilities to specify xtics, ytics and ztics
- Set horizontal rather than vertical layout for `Line.line`
- Simple box plots
- Remove useless `plot_matrix` type
- Remove `plot` and labelled argument for `run`
- Add `tup_r2, tup_r3`

## 0.0.1
- First release

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/prbnmcn-gnuplot/prbnmcn-gnuplot.0.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+description: "Declarative generation of gnuplot scripts"
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "MIT"
+homepage: "http://github.com/igarnier/prbnmcn-gnuplot"
+doc: "https://igarnier.github.io/prbnmcn-gnuplot"
+bug-reports: "http://github.com/igarnier/prbnmcn-gnuplot"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08.0"}
```
Comment (kit-ty-kate) [inline packages/prbnmcn-gnuplot/prbnmcn-gnuplot.0.0.5/opam:13]: I'm not sure the reason behind that, but the tests fail on OCaml 4.12 and older
```suggestion
  "ocaml" {>= "4.13.0"}
```

File: packages/prbnmcn-gnuplot/prbnmcn-gnuplot.0.0.5/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "base-unix"
+  "conf-gnuplot" {with-test}
+  "odoc" {with-doc}
+]
+x-ci-accept-failures: [
+  "centos-7" # gnuplot too old
+  "oraclelinux-7" # gnuplot too old
```
Comment (kit-ty-kate) [inline packages/prbnmcn-gnuplot/prbnmcn-gnuplot.0.0.5/opam:20]:
```suggestion
  "oraclelinux-7" # gnuplot too old
  "debian-10" # gnuplot too old
  "oraclelinux-8" # gnuplot too old
  "ubuntu-18.04" # gnuplot too old
  "ubuntu-20.04" # gnuplot too old
```

Comment (igarnier) [reply packages/prbnmcn-gnuplot/prbnmcn-gnuplot.0.0.5/opam:20]: Thanks a lot, I applied your suggestion.

**Comment by igarnier** [comment]
I deactivated the tests, short of a better solution. There's still some error of the form "fatal: cannot create directory at 'packages/doc-ock/doc-ock.1.2.0': No space left on device" but I don't think it's related to the package.

**Comment by kit-ty-kate** [comment]
Thanks

**Comment by igarnier** [comment]
No worries, thank you!

**Pull request merged**